modules/client-configuration/pages/cobbler.adoc

```diff
@@ -53,9 +55,10 @@ A sync is basically a rebuild of every file cobbler touched. I will try give you
 To use Cobbler for system installation with PXE, you will require a TFTP server.
 {productname} installs a TFTP server by default.
 To PXE boot systems, you will require a DHCP server, or have access to a network DHCP server.
+// ke, 2019-11-26: Maybe, we should say that this on SUMA has to be the address of the SUMA server?
 Edit the [path]``/etc/dhcp.conf`` configuration file to change [option]``next-server`` to the hostname or IP address of your Cobbler server.
 
-Cobbler requires an open HTTP port to synchronize data between the Server and the Proxy.
+Cobbler requires an open HTTP port to synchronize data between the server and the proxy.
```

Comment:
Question: Do we explain tftpsync somewhere else ? Because cobbler itself does not require these ports to be open. At least not for the features of cobbler we use.

But we have such a sync mechanism implemented as a trigger of cobbler. This is tftpsync. You need to explicitly configure it. So if you do this, you need to open the ports. But by default the ports are not needed.

On the other hand: "HTTP(s)" is used for all the other features of SUSE Manager Server and Proxy. Closing them, completly makes the product unusable.

But in this context here we can remove it if we have a chapter about tftpsync.

Reply:
Just found it here in the same page. 

```
You can also synchronize Cobbler-generated TFTP contents to a SUSE Manager Proxy.
...
```
Here we can add the note, that the sync happens via https and the ports needs to be open.

Reply:
ok, I moved this info.  thanks!